README.md

```diff
@@ -291,6 +291,25 @@ try {
 }
 ```
 
+To enable caching for the requests, one can add `expiresIn` to the requests configuration object. The values are in seconds.
+```
+// cache is valid for 30 minutes
+const requestsConfig = {
+  cache: {
+  expiresIn: 1800,
+  }
+};
+```
+
+To disbale caching for the requests that are enbaled by default, you can assign 0 to expiresIn.
```

Comment:
Typos: `disbale`, `enbaled`.

I would also suggest putting "expiresIn" in ticks (`expiresIn` instead of expiresIn).

Reply:
Fixed